.github/workflows/testing.yml

```diff
@@ -73,15 +72,21 @@ jobs:
           python -m pip install --upgrade -r dev-requirements.txt
           python -m pip install --upgrade .
       - name: Try single CLI run of tool
+        env:
+          nvd_api_key: ${{ secrets.NVD_API_KEY }}
```

Comment:
```suggestion
```

We should be able to skip passing the key here, since we're explicitly calling with `-n json` so it's not getting used.  (I still don't have an answer for how this will be handled once the json method goes away, but we've got a year to figure that out.)